client/src/modules/stock/aggregated_consumption/aggregated_consumption.js

```diff
@@ -145,6 +150,11 @@ function StockAggregatedConsumptionController(
       displayName : 'TABLE.COLUMNS.CONSUMPTION',
       headerCellFilter : 'translate',
       cellTemplate : 'modules/stock/aggregated_consumption/templates/lot_aggregate.tmpl.html',
+    }, {
```

Comment:
These look good!  I'm going to put `text-right` on them so they can be summed easily in a user's head.